torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -69,3 +69,19 @@ void flag_to_float(const T* src, float* dst, int64_t n) {
     dst_u32[i] = *(src + i) ? 0xFFFFFFFF : 0;
   }
 }
+
+#if defined(CPU_CAPABILITY_AVX512)
+
+#define TILE2D_COPY_TRANSPOSE(dst, src, ld_dst, ld_src)       \
```

Comment:
Why do we need these macros?  Perhaps call at::vec directly?

Reply:
Thanks for the comment. I intended to let the generated code invoke something with general name. Now I reverted this and added the general `transpose_mxn` template function into at::vec. Please review. Thanks.